react/features/external-api/middleware.js

```diff
@@ -88,7 +88,9 @@ MiddlewareRegistry.register(store => next => action => {
     case CONFERENCE_JOINED: {
         const state = store.getState();
         const { defaultLocalDisplayName } = state['features/base/config'];
+        const { room } = state['features/base/conference'];
         const { loadableAvatarUrl, name, id } = getLocalParticipant(state);
+        const breakoutRoom = APP.conference.roomName.toString() !== room.toLowerCase();
```

Comment:
can room name be something else than a string? also, is it always lowercase? I think so, but just to make sure.

Reply:
From what I've tested, it can be a String instance and it's case insensitive.